dev/conductor/core/lib/src/packages_autoroller.dart

```diff
@@ -0,0 +1,263 @@
+        'auth',
+        'login',
+        '--hostname',
+        hostname,
+        '--git-protocol',
+        'https',
+        '--with-token',
+      ],
+      stdin: token,
+    );
+  }
+
+  /// Create a pull request on GitHub.
+  ///
+  /// When the current branch isn't fully pushed to a git remote, a prompt will ask where
+  /// to push the branch and offer an option to fork the base repository. Use `--head` to
+  /// explicitly skip any forking or pushing behavior.
+  ///
+  /// A prompt will also ask for the title and the body of the pull request. Use `--title`
+  /// and `--body` to skip this, or use `--fill` to autofill these values from git commits.
+  ///
+  /// Link an issue to the pull request by referencing the issue in the body of the pull
+  /// request. If the body text mentions `Fixes #123` or `Closes #123`, the referenced issue
+  /// will automatically get closed when the pull request gets merged.
+  ///
+  /// By default, users with write access to the base repository can push new commits to the
+  /// head branch of the pull request. Disable this with `--no-maintainer-edit`.
+  ///
+  ///
+  /// USAGE
+  ///   gh pr create [flags]
+  ///
+  /// FLAGS
+  ///   -a, --assignee login       Assign people by their login. Use "@me" to self-assign.
+  ///   -B, --base branch          The branch into which you want your code merged
+  ///   -b, --body string          Body for the pull request
+  ///   -F, --body-file file       Read body text from file
+  ///   -d, --draft                Mark pull request as a draft
+  ///   -f, --fill                 Do not prompt for title/body and just use commit info
+  ///   -H, --head branch          The branch that contains commits for your pull request (default: current branch)
+  ///   -l, --label name           Add labels by name
+  ///   -m, --milestone name       Add the pull request to a milestone by name
+  ///       --no-maintainer-edit   Disable maintainer's ability to modify pull request
+  ///   -p, --project name         Add the pull request to projects by name
+  ///       --recover string       Recover input from a failed run of create
+  ///   -r, --reviewer handle      Request reviews from people or teams by their handle
+  ///   -t, --title string         Title for the pull request
+  ///   -w, --web                  Open the web browser to create a pull request
+  ///
+  /// INHERITED FLAGS
+  ///       --help                     Show help for command
+  ///   -R, --repo [HOST/]OWNER/REPO   Select another repository using the [HOST/]OWNER/REPO format
+  ///
+  /// EXAMPLES
+  ///   $ gh pr create --title "The bug is fixed" --body "Everything works again"
+  ///   $ gh pr create --reviewer monalisa,hubot  --reviewer myorg/team-name
+  ///   $ gh pr create --project "Roadmap"
+  ///   $ gh pr create --base develop --head monalisa:feature
+  Future<void> createPr({
+    required io.Directory repository,
+    String title = '[flutter_tools] Auto-roll pub packages',
```

Comment:
Should the hashtag be generalized since this will impact more than the tools?

Reply:
Hmm, you're right, in this theoretically affects everything in the repo. Should I just leave out the namespace? AFAIK, nothing parses this except my own GitHub queries come perf time.

Reply:
Removed the namespace